### PR TITLE
add ticket requirements.

### PR DESCRIPTION
**Pull request by schangccg**
for ticket: https://github.com/muccg/bpa-archive-ops/issues/581

### CODE REVIEW

File: bpaotu/bpaotu/static/bpaotu/bpaotu.js [hunk 1]
```diff
@@ -577,10 +577,22 @@ $(document).ready(function() {
             async: true,
             success: function(result) {
                 authentication_token = result;
+
                 run_setup_functions();
+
+                var data_portion = $.parseJSON(authentication_token.split("||")[1]);
+                var username = (data_portion.email).split("@")[0];
```
Comment (grahame) [inline bpaotu/bpaotu/static/bpaotu/bpaotu.js]: in retrospect I think this'll be confusing -let's just show the full email address

File: bpaotu/bpaotu/static/bpaotu/table.js [hunk 2]
```diff
@@ -520,9 +520,19 @@ $(document).ready(function() {
             success: function(result) {
                 authentication_token = result;
                 run_setup_functions();
+
+                var data_portion = $.parseJSON(authentication_token.split("||")[1]);
+                var username = (data_portion.email).split("@")[0];
+                var user_string = username + " | " + '<a href="/user/_logout">Sign Out</a>';
+
+                $("#user_id").html(user_string);
```
Comment (grahame) [inline bpaotu/bpaotu/static/bpaotu/table.js]: don't do this - what if the username somehow has HTML in it? use jQuery to safely build DOM elements (check the jQuery docs)

File: bpaotu/bpaotu/static/bpaotu/bpaotu.js (hunk 1 above)
Comment (schangccg) [reply bpaotu/bpaotu/static/bpaotu/bpaotu.js]: Fixed.

File: bpaotu/bpaotu/static/bpaotu/table.js (hunk 2 above)
Comment (schangccg) [reply bpaotu/bpaotu/static/bpaotu/table.js]: Fixed.

**Pull request merged**